src/makeCalculator.js

```diff
@@ -38,6 +38,32 @@
  */
 function makeCalculator() {
   // write code here
+  return {
+    result: 0,
+
+    add(a, b) {
+      return a + b;
+    },
+    subtract(a, b) {
+      return a - b;
+    },
+    multiply(a, b) {
+      return a * b;
+    },
+    divide(a, b) {
+      return a / b;
```

Comment:
The 'divide' method should handle the case when 'b' is zero to prevent division by zero error.